src/main/java/uk/gov/companieshouse/pscfiling/api/controller/impl/ValidationStatusControllerImpl.java

```diff
@@ -37,6 +36,7 @@
 public class ValidationStatusControllerImpl implements ValidationStatusController {
     public static final String TRANSACTION_NOT_SUPPORTED_ERROR =
             "Transaction not supported: FEATURE_FLAG_TRANSACTIONS_CLOSABLE=false";
+    public static final String SERVICE_UNAVAILABLE_ERROR = "Service Unavailable: {error}";
```

Comment:
What is this used for? Don't see it referenced.

Reply:
being used in:
src/test/java/uk/gov/companieshouse/pscfiling/api/controller/impl/PscIndividualFilingControllerImplIT.java
src/test/java/uk/gov/companieshouse/pscfiling/api/controller/impl/PscWithIdentificationFilingControllerImplIT.java
src/test/java/uk/gov/companieshouse/pscfiling/api/controller/impl/ValidationStatusControllerImplFlagUndefinedIT.java

Reply:
Yeah just saw it :)